### ISSUE TITLE
[1.12+] Electric engine doesn't update configuration properly

### ISSUE THREAD

**Issue by OneEyeMaker** [ForestryMC/ForestryMC#1907]
Hello.
I found strange issue. Electric engine doesn't update it's configuration (installed circuit) properly.
Steps to reproduce:
1) Place Electric engine and connect it to EU source and RF output (RTG from IC2 and Creative RF cell in my case).
2) Install 4 Bronze electron tubes into Intricate circuit board with configuration 'Electric engine'.
3) Install this circuit board into Electric engine (energy output increases from 20 RF/t to 180 RF/t).
4) Remove circuit board from Electric engine via Soldering iron.

After last step output of Electric engine **doesn't change** and stays **high** (180 RF/t). Energy consumption also stays high (I checked it with EU Reader).
In other words, Electric engine doesn't update it's configuration after removing circuit board.

MC 1.12.2
Forge build 2560
Forestry 5.7.0.214
IC2 2.8.16-ex112
